templates/template_flow.py

```diff
@@ -0,0 +1,110 @@
+import os
+
+from metaflow import (
+    FlowSpec, IncludeFile, Parameter,
+    card, current, step,
+    environment, kubernetes
+)
+from metaflow.cards import Markdown
+
+from artifact_store import ArtifactStore
+
+GCS_PROJECT_NAME = "project-name-here"
+GCS_BUCKET_NAME = "bucket-name-here"
+
+class TemplateFlow(FlowSpec):
+    """
+    This flow is a template for you to use
+    for orchestration of your model.
+    """
+    example_config = IncludeFile("example_config", default="./example_config.json")
```

Comment:
Let's add a comment explaining the purpose of this line: this is an example of how you make a file available to your flow. In this case, a configuration. This is only meant for small files like configuration files.

Reply:
can do!